test/BackportAssertionsTrait.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace LaminasTest\ServiceManager;
+
+use PHPUnit\Framework\Constraint\IsType;
+use PHPUnit\Framework\ExpectationFailedException;
+
+/**
+ * Backport Assertion Already removed latest PHPUnit.
+ *
+ * @see https://github.com/sebastianbergmann/phpunit/issues/3339
+ *
+ * borrowed from
+ * https://github.com/sebastianbergmann/phpunit/blob/7.5.9/src/Framework/Assert.php#L1408
+ */
+trait BackportAssertionsTrait
```

Comment:
The actual problem is only moved but not fixed.
Please update the related tests instead of introducing new code which must be maintained additionally.

There is a moderate number of tests:

* https://github.com/laminas/laminas-servicemanager/search?q=assertAttributeSame
* https://github.com/laminas/laminas-servicemanager/search?q=assertAttributeEquals
* https://github.com/laminas/laminas-servicemanager/search?q=assertInternalType

Thanks in advance!

Reply:
I have replaced them. 

- assertInternalType => assertIsArrar.
- for AssertEquals  I have added getCreationContext() inside the TestAssert.
- for assertAttributeSame I have added anonymous classes to get access to the private properties.